requirements.txt

```diff
@@ -0,0 +1,2 @@
+pyjwt==1.5.2
+requests>=2.11.0
```

Comment:
I actually ran into a problem with this (and as you can see in commit history I removed the `requirements.in`)... `requirements.in` fixes the library version of requests to X in `requirements.txt` (even with `>=`). This then causes a conflict in MM because `edx-api-client` depends on version Y and `pip-compile` fails to resolve a valid version that matches both X and Y in MM. Specifying a range allows `pip-compile` to resolve.

Reply:
ah gotcha, makes sense.

Reply:
yeah, honestly, `edx-api-client` probably needs to be fixed in that regard too